.gitignore

```diff
@@ -0,0 +1,2 @@
+coverage*.out
+.sysl-tmp/*
```

Comment:
I dont think this is needed. There is nothing that creates `.sysl-tmp` that I can see in the sysl repo or this one.

Reply:
Good point, I've fixed this up in 8ada55f0ab451ef08f4edf2366310467fa084480 (will be doing a squash merge)

Reply:
It came back into mine, but until someone can justify it I guess it's OK to remove.